libs/core/models/broadcasted-time.model.ts

```diff
@@ -0,0 +1,19 @@
+import { Immerable, OmitImmerable } from './immerable';
+
+/** Genre. */
+export class BroadCastedTime extends Immerable {
+
+	/** An anime can have a started date of date or null if it's not decide yet. */
+	public startDate: Date | null;
+
+	/** An anime can have a end date if it ends already or null if it not end yet. */
+	public endDate: Date | null;
```

Comment:
Properties should be readonly (see our [wiki](https://wiki.saritasa.rocks/frontend/languages/ts-js/best-practices#interfaces)):
```suggestion
	public readonly startDate: Date | null;

	/** An anime can have a end date if it ends already or null if it not end yet. */
	public readonly endDate: Date | null;
```